examples/simple-jsbeautifyrc/tsx/expected/test.tsx

```diff
@@ -1,12 +1,10 @@
 class Test extends React.Component<Foo> {
   render() {
-    return (
-      <div className= "class" >
-      <h2 className="anotherClass" >
-        { this.foo.bar }
-        < /h2>
-    { this.foo.bar.children }
-    </div>
-    );
+    return (<div className="class">
```

Comment:
Concerned about switching from newline to no-newline here.
Is there a configuration option for this?